### PR TITLE
Add a user guide

### PR DESCRIPTION
**Pull request by font**
This addresses #84.

cc @quinton-hoole @pmorie

### CODE REVIEW

**Comment by quinton-hoole** [comment]
I can review this.  Is it expected that I could step through this guide as is, and federation-v2 should work?  If so, I will try to do that, and report any issues I find.

**Comment by font** [comment]
@quinton-hoole Yep, that is the expectation. You can now go through it and report back any issues.

File: docs/userguide.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+```bash
+for tc in ./config/federatedtypes/*.yaml; do kubectl create -f "${tc}"; done
+```
+
+Once this is complete, you now have a working federation-v2 control plane and
+can proceed to join clusters.
+
+### Join Clusters
+
+Next, you'll want to use the `kubefnord` tool to join all your
```
Comment (quinton-hoole) [inline docs/userguide.md]: Lets give this tool a better name.  
I've opened https://github.com/kubernetes-sigs/federation-v2/issues/103 to track that.
We can merge this doc as is, and update it once the above is done.

**Comment by quinton-hoole** [comment]
LGTM
I haven't yet had a chance to verify that each step works correctly, but will do soon.  The doc looks good though.  Separate PR's can be submitted to fix anything that's broken.

I'll leave this open for a few days for anyone else who wants to review it, and then we can merge.

**Comment by pmorie** [comment]
I'm about to try these steps manually, so I'll make updates as necessary in their own PRs.

**Pull request merged**